indico/modules/events/registration/templates/display/_registration_summary_blocks.html

```diff
@@ -141,11 +141,20 @@
                             <td class="text-right">{{ item.price }} {{ currency }}</td>
                         </tr>
                     {% endfor %}
+                    {% if registration.registration_form.base_price %}
+                        <tr>
```

Comment:
I think you have to add the `class="regform-done-table-item"` otherwise this row will not follow the row highlighting of the table.